spec/requests/submissions_spec.rb

```diff
@@ -233,5 +233,65 @@
         end
       end
     end
+
+    context 'when paginating submissions' do
+      let!(:submissions) do
+        (1..25).map do |n|
+          create(:submission, challenge: challenge, phase: phase)
+        end
+      end
+
+      it 'returns first page of submissions' do
+        get submissions_phase_path(phase)
+        expect(response.body).to have_css('tr[data-submission-id]', count: 20)
```

Comment:
want to add an expectation that the "Load more" button exists and is hidden when there is no more data?